packages/react/tabs/src/Tabs.stories.tsx

```diff
@@ -253,6 +349,7 @@ const RECOMMENDED_CSS__TABS__ROOT = {
 
 const rootClass = css({
   ...RECOMMENDED_CSS__TABS__ROOT,
+  border: '1px solid #eee',
```

Comment:
@benoitgrelard I remembered that the diff change is because I moved this border to the outer wrapper so that the animations look better in the story.